### PR TITLE
Adding editorconfig for current settings

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# top-most EditorConfig file
+root = true
+
+# settings that apply to all text files
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.{js,json,css,scss,mustache,html}]
```
Comment (nemtsov) [inline .editorconfig]: Should we add `.md` here?

**Comment by nemtsov** [comment]
👍

File: .editorconfig (hunk 1 above)
Comment (weikinhuang) [inline .editorconfig]: we weirdly have a indent of `4` on certain project's `.md` files so i'm just keeping it consistient

Comment (nemtsov) [inline .editorconfig]: Hmm. I guess my question is then whether we want to perpetuate that style or flag it and slowly move to two spaces for the .md files

**Comment by kaicataldo** [comment]
:+1:

**Pull request merged**